src/Foundation/Rfc3339.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAPIValidation\Foundation;
+
+use DateTime;
+use DateTimeZone;
+use function preg_match;
+use function strpos;
+use function strtoupper;
+
+class Rfc3339
```

Comment:
As an ordinary user this naming would confuse me until I dig into it to understarnd what it does. I suggest name it something like `Rfc3339DatetimeFactory`